assets/css/structure.css

```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: var(--primary-font);
+  background-color: var(--primary-color);
+  place-items: center;
+}
+
+.site-wrapper {
+  max-width: 1200px;
+  padding: 0 var(--spacing-small);
+  margin: 0 auto;
+}
+
+.content {
+  height: 100vh;
```

Comment:
this is making issue of space between your elements. remove it from here.

Reply:
Fixed.